.github/dependabot.yml

```diff
@@ -4,23 +4,31 @@
 #
 version: 2
 updates:
-- package-ecosystem: gomod
+- package-ecosystem: docker
   directory: "/"
   schedule:
     interval: daily
     time: "04:00"
-  open-pull-requests-limit: 10
+  target-branch: master
   reviewers:
   - giantswarm/team-firecracker-engineers
-  ignore:
-  - dependency-name: k8s.io/*
-    versions:
-    - ">=0.17.0"
-- package-ecosystem: docker
+- package-ecosystem: github-actions
   directory: "/"
   schedule:
     interval: daily
     time: "04:00"
   target-branch: master
   reviewers:
   - giantswarm/team-firecracker-engineers
+- package-ecosystem: gomod
+  directory: "/"
+  schedule:
+    interval: daily
+    time: "04:00"
+  open-pull-requests-limit: 10
+  reviewers:
+  - giantswarm/team-firecracker-engineers
+  ignore:
+  - dependency-name: k8s.io/*
+    versions:
+    - ">=0.19.0"
```

Comment:
Also this here should ensure that we actually get patch updates for our currently used k8s client libs. If this breaks anything we can revert again.